sourcegraph/sourcegraph
# user-management: add table column filtering

Closes https://github.com/sourcegraph/sourcegraph/issues/40593, https://github.com/sourcegraph/sourcegraph/issues/40944.
Blocked by https://github.com/sourcegraph/sourcegraph/pull/40679.

## Test plan
- `sg start`
- Open http://localhost:3080/site-admin/users
- Check that table column filters work correctly and all synced with URL state

## Screenshots
| Description | Screenshot |
| --: | :-- |
| Overview | <img width="992" alt="image" src="https://user-images.githubusercontent.com/6717049/186946630-578426c4-e03e-41f8-a417-53534727b39a.png"> |
| Date range with negation filter |  <img width="992" alt="image" src="https://user-images.githubusercontent.com/6717049/186946705-3e7e4e5a-6f95-48d3-92f7-1baea142a995.png"> |
|  | <img width="1094" alt="image" src="https://user-images.githubusercontent.com/6717049/186946866-6d627b8e-4405-47f3-8b9c-2f2d32838a04.png"> |
|  | <img width="1180" alt="image" src="https://user-images.githubusercontent.com/6717049/186946917-52580e78-b0fb-482d-82fb-98cc4aa274a9.png"> |
